cloud_level_testing/features/steps/tools.py

```diff
@@ -181,62 +185,65 @@ def collect_ips(client,logger:Logger):
         assertline = f"No ips found"
     return ips, assertline
 
-def collect_jhs(client, test_name,logger:Logger):
+
+def collect_jhs(client, test_name, logger: Logger):
     servers = client.compute.servers()
-    lookup = test_name+"-jh"
-    lookup = "default-jh" # just for testing delete later
+    lookup = test_name + "-jh"
```

Comment:
please use fstring 

lookup = f"{test_name}-jh"

Reply:
This is not from made by me but i will change it.